draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -4236,6 +4240,57 @@ Content-Range: exampleunit 11.2-14.3/25
    does not prevent such a mechanism from being developed as an extension.
 </t>
 </section>
+
+<section title="Request Payload Negotiation" anchor="request.payload.negotiation">
+  <x:anchor-alias value="request payload negotiation"/>
+<t>
+   When content negotiation preferences are sent in a server's response, the
+   listed preferences are called <x:dfn>request payload negotiation</x:dfn>
+   because they intend to influence selection of an appropriate payload for
+   subsequent requests to that resource. For example,
+   the <x:ref>Accept-Encoding</x:ref> field
+   (<xref target="header.accept-encoding"/>) can be sent in a response to
+   indicate preferred content codings for subsequent requests to that
+   resource <xref target="RFC7694"/>.
+</t>
+<aside>
+  <t>
+    Similarly, <xref target="RFC5789" x:sec="3.1" x:fmt="of"/> defines
+    the "Accept-Patch" response header field which allows discovery of
+    which content types are accepted in PATCH requests.
+  </t>
+</aside>
+</section>
+
+<section title="Quality Values" anchor="quality.values">
+  <x:anchor-alias value="weight"/>
+  <x:anchor-alias value="qvalue"/>
+<t>
+   The content negotiation fields
```

Comment:
Either "The content negotiation fields defined by this specification" or "some content negotiation fields."

Reply:
done